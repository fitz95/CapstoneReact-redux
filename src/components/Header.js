import React from 'react';
import '../css/header.css';
import allcoins from '../images/pngegg.png';

function Header() {
  return (
    <div className="headerdiv">
      <img alt="cryptocurrencies" src={allcoins} className="imgheader" />
      <div className="subdiv">
        <h1 className="headerh1">CRYPTO MARKET VALUE</h1>
        <p className="headerh2">100 coins</p>
      </div>
    </div>
  );
}

export default Header;
