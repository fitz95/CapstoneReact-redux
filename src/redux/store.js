import { configureStore, combineReducers } from '@reduxjs/toolkit';
import logger from 'redux-logger';
import coinsReducer from './coins/coinsSlice';
import coinDetailsReducer from './coindetails/coinDetailsSlice';

const store = configureStore({
  reducer: combineReducers({
    coins: coinsReducer,
    coinDetails: coinDetailsReducer,
  }),
  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});

export default store;
