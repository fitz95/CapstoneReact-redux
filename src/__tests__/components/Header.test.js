import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from 'src/redux/store';
import Header from 'src/components/Header';

describe('RocketsLists', () => {
  it('renders correctly for reserved state', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Header />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
