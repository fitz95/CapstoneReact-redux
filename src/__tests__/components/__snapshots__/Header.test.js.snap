// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RocketsLists renders correctly for reserved state 1`] = `
<div
  className="headerdiv"
>
  <img
    alt="cryptocurrencies"
    className="imgheader"
    src="pngegg.png"
  />
  <div
    className="subdiv"
  >
    <h1
      className="headerh1"
    >
      CRYPTO MARKET VALUE
    </h1>
    <p
      className="headerh2"
    >
      100 coins
    </p>
  </div>
</div>
`;
